techlottus/portalverse-uane
# [DEV-5241] Migrate "oferta educativa" pages images

## Issue
 - Migrate images of "oferta educativa" paseg from drive to S3 bucket

## Solution
- [feat: Bachillerato page images migrated](https://github.com/techlottus/portalverse-uane/pull/123/commits/6043edfbb6c1dc009623dd9b32e2d443b19b7bba)
- [feat: Licenciaturas page images migrated](https://github.com/techlottus/portalverse-uane/pull/123/commits/3694ff83bccd61f4fcc555ef662d3caebf619217)
- [feat: Especialidades page images migrated](https://github.com/techlottus/portalverse-uane/pull/123/commits/ade2198adc2386bc35a09e66a0ac9dfea32ac131)
- [feat: Maestrias page images migrated](https://github.com/techlottus/portalverse-uane/pull/123/commits/045961dee9f1961715a4f1308136a139fe91671b)
- [feat: Doctorado page images migrated](https://github.com/techlottus/portalverse-uane/pull/123/commits/f181d3fa84555f145b7a7b03564a75883f660cf8)
- [feat: extensión universitaria page images migrated](https://github.com/techlottus/portalverse-uane/pull/123/commits/77e161d73d41bb5d0224d1a872c37aa1656f4725)
- [feat: licenciaturas images migrated](https://github.com/techlottus/portalverse-uane/pull/123/commits/2d3274669d6d9632617e1d8d49e3b7fccef95440)
- [feat: masters images migrated](https://github.com/techlottus/portalverse-uane/pull/123/commits/3a2c37eb739bda55491fab077df1fc0295da4a38)
- [feat: doctorados images migrated](https://github.com/techlottus/portalverse-uane/pull/123/commits/7c9d886215eb4ec823a19b9bffa133692bb880bc)
- [feat: update licenciatura image](https://github.com/techlottus/portalverse-uane/pull/123/commits/ef7b6d87021a2deefe88524773a20d802947af6d)

## Testing
 - Go to `/oferta-educativa`
 - Click on any section 
![image](https://user-images.githubusercontent.com/62124025/232914676-8f22cc70-e028-4d94-8213-b88f680cc4d9.png)
 - Compare with [UANE](https://uane.edu.mx/oferta-educativa) website
 - Check the URL of images (assets.staging.bedu.org)
 - Keep rocking 🔥
